src/framework/audio/common/rpc/contextrpcchannel.cpp

```
@@ -0,0 +1,69 @@
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#include "contextrpcchannel.h"
+
+using namespace muse::audio::rpc;
+
+void ContextRpcChannel::send(const Msg& msg, const Handler& onResponse)
+{
```

Cppcheck (2.20.0)

[information] Include file

(missingInclude)